app/utils/newsFeed/redis/mappings.ts

```diff
@@ -298,3 +305,24 @@ export const mapToRedisUser = async (userId: string): Promise<RedisUser> => {
   const user = await getInnoUserByProviderId(userId);
   return user;
 };
+
+const updateImageUrl = (imageUrl: string | undefined): string | undefined => {
+  if (imageUrl && imageUrl.startsWith('http://127.0.0.1:1337/')) {
```

Comment:
Instead of just replacing strapi URLs staring with `http://127.0.0.1...` with their relative paths, we want to replace all URLs with their relative paths. In my dev environment for example, the strapi URL is `http://localhost:1337`. Other deployed instances (e.g. for beta) will have other URLs as well.

Try extracting the path name like this:
- First check if the image URL is a valid URL (you can use `URL.canParse(imageUrl, clientConfig.NEXT_PUBLIC_STRAPI_ENDPOINT)` for this
- If it is a valid URL, you can extract the path in the URL using the `URL` class:  
```
const url = new Url(imageUrl)
const path = url.pathname
```